CHANGELOG.md

```diff
@@ -258,7 +260,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 
 
-[Unreleased]: https://github.com/giantswarm/azure-operator/compare/v5.0.0-beta5...HEAD
+[Unreleased]: https://github.com/giantswarm/azure-operator/compare/v5.1.0...HEAD
+[5.1.0]: https://github.com/giantswarm/azure-operator/compare/v5.0.0-beta5...v5.1.0
 [5.0.0-beta5]: https://github.com/giantswarm/azure-operator/compare/v5.0.0-beta4...v5.0.0-beta5
```

Comment:
Should we merge the `release-5.0.x` CHANGELOG into master?

Reply:
makese sense to me

Reply:
Ok... I did  this now manually.